base/applications/rapps/available.cpp

```diff
@@ -38,6 +38,13 @@ VOID CAvailableApplicationInfo::RetrieveGeneralInfo(AvailableStrings& AvlbString
 {
     m_Parser = new CConfigParser(m_sFileName);
 
+    // TODO: I temporarily use the file name (without suffix) as package name.
+    // It should be better to put this in a field of ini file.
+    // consider write a converter to do this and write a github action for rapps-db to ensure package_name is unique.
```

Comment:
As noted before, I am in favor of keeping this the filename, and only adding a key when needed. :)

Reply:
So I left it here🤣
but I think adding ```m_szPkgName``` is still needed (as it will be used more than once and I don't want to parse the filename to remove suffix again and again😝)